x-pack/test/security_solution_cypress/cypress/support/saml_auth.ts

```diff
@@ -67,13 +67,13 @@ export const samlAuthentication = async (
     cloudUsersFilePath,
```

Comment:
let's leverage available typings from

```
import type { Role } from '@kbn/security-plugin-types-common';
```


I posted a commit suggestions, that would work when line above will be added to this file